src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -625,8 +625,8 @@ export default class FilingHistoryList extends Vue {
         }
       }
 
-      // add properties for Dissolutions
-      if (this.isTypeDissolution(filing)) {
+      // add properties for Voluntary Dissolutions
+      if (this.isTypeVoluntaryDissolution(filing)) {
```

Comment:
I'm calling this a "fix" because, if this was an administrative dissolution, then the old code would execute this block as well as the block at line 668 below.

THIS block is specific to voluntary dissolutions only.